controllers/protectedvolumereplicationgrouplist_controller.go

```diff
@@ -26,6 +28,7 @@ type ProtectedVolumeReplicationGroupListReconciler struct {
 	APIReader      client.Reader
 	ObjStoreGetter ObjectStoreGetter
 	Scheme         *runtime.Scheme
+	RateLimiter    *workqueue.RateLimiter
```

Comment:
This is used only during tests, and otherwise we use the default rate limiter?

Reply:
yes